src/remove-search-spec.js

```diff
@@ -2,21 +2,22 @@
 
 const fs = require('fs-extra');
 const path = require('path');
+const utils = require('./utils/shared');
 
 function removeSearchSpecFromProject(argv, config) {
-  if (config &&
-      config.appSettings &&
-      config.appSettings.stache &&
-      config.appSettings.stache.searchConfig &&
-      config.appSettings.stache.searchConfig.allowSiteToBeSearched !== false) {
-    try {
-      let filePath = path.join(process.cwd(), 'e2e', 'stache-search.e2e-spec.ts');
-      if (fs.existsSync(filePath)) {
-        fs.unlinkSync(filePath);
-      }
-    } catch (error) {
-      throw new Error('[ERROR]: Unable to remove stache search template from e2e directory.');
+  let doesSearchConfigExist = utils.checkConfig(config, 'allowSiteToBeSearched');
```

Comment:
Any thought on instead having checkConfig be more of a "safe read?"  Something like...
```
if (utils.readConfig(config, 'allowSiteToBeSearch') === false) {
...
}

function readConfig(config, key) {
  if (config &&
  config.appSettings &&
  config.appSettings.stache &&
  config.appSettings.stache.searchConfig) {
    return config.appSettings.stache.searchConfig[key];
  }

  // Not found, "returns" undefined
}
```